connect/runtime/src/test/java/org/apache/kafka/connect/runtime/rest/resources/ConnectorsResourceTest.java

```diff
@@ -519,6 +519,38 @@ public void testGetConnectorConfigConnectorNotFound() throws Throwable {
         PowerMock.verifyAll();
     }
 
+    @Test
+    public void testGetTasksConfig() throws Throwable {
```

Comment:
Should we add one more test to test multiple tasks with multiple config properties?